ksql-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -1034,6 +1039,61 @@ public void shouldPrintWarnings() throws IOException {
     }
   }
 
+  @Test
+  public void shouldPrintErrorEntityLongNonJson() throws IOException {
+    // Given:
+    final KsqlEntity entity = new ErrorEntity(
+        "statementText",
+        Strings.repeat("Not a JSON value! ", 10));
+
+    // When:
+    console.printKsqlEntityList(ImmutableList.of(entity));
+
+    // Then:
+    final String output = terminal.getOutputString();
+    if (console.getOutputFormat() == OutputFormat.TABULAR) {
```

Comment:
shouldn't this be an assertion? What if we have a bug where the output format is not TABULAR? This test will never fail.

Reply:
JSON is for the REST API, tabular is what the CLI displays - I think we need both. This test doesn't make sense for the JSON case because we don't do anything special for it (it's testing formatting in the CLI)

Reply:
I see. I didn't notice this test is parameterized with JSON or TABULAR.